app/interactions/matches/create_match.rb

```diff
@@ -0,0 +1,178 @@
+    Footman.create!(player: player, board: match.board, territory: "kingswood", x: 1098, y: 2000)
+    player
+  end
+
+  def create_lannister_player(match)
+    player = Player.create!(match: match, house: "Lannister")
+    Boat.create!(player: player, board: match.board, territory: "the_golden_sound", x: 133, y: 1759)
+    Knight.create!(player: player, board: match.board, territory: "lannisport", x: 328, y: 1826)
+    Footman.create!(player: player, board: match.board, territory: "lannisport", x: 348, y: 1690)
+    Footman.create!(player: player, board: match.board, territory: "stoney_sept", x: 476, y: 1820)
+    player
+  end
+
+  def create_stark_player(match)
+    player = Player.create!(match: match, house: "Stark")
+    board = match.board
+    Boat.create!(player: player, board: board, territory: "the_shivering_sea", x: 1208, y: 738)
+    Knight.create!(player: player, board: board, territory: "winterfell", x: 625, y: 851)
+    Footman.create!(player: player, board: board, territory: "winterfell", x: 773, y: 544)
+    Footman.create!(player: player, board: board, territory: "white_harbor", x: 852, y: 810)
+    player
+  end
+
+  def create_tyrell_player(match)
+    player = Player.create!(match: match, house: "Tyrell")
+    Boat.create!(player: player, board: match.board, territory: "redwyne_straights", x: 77, y: 2504)
+    Knight.create!(player: player, board: match.board, territory: "highgarden", x: 389, y: 2171)
+    Footman.create!(player: player, board: match.board, territory: "highgarden", x: 406, y: 2262)
+    Footman.create!(player: player, board: match.board, territory: "dornish_marches", x: 498, y: 2308)
+    player
+  end
+
+  def create_martell_player(match)
+    player = Player.create!(match: match, house: "Martell")
+    Boat.create!(player: player, board: match.board, territory: "sea_of_dorne", x: 1121, y: 2475)
+    Knight.create!(player: player, board: match.board, territory: "sunspear", x: 1010, y: 2600)
+    Footman.create!(player: player, board: match.board, territory: "sunspear", x: 1107, y: 2611)
+    Footman.create!(player: player, board: match.board, territory: "salt_shore", x: 814, y: 2698)
+    player
+  end
+
+  def create_tracks(match, board, players)
+    [
+      WildlingsTrack,
+      VictoryTrack,
+      # SupplyTrack,               # TODO
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l118)